tests/accel/upgrade/tests/workload.go

```diff
@@ -0,0 +1,215 @@
+package upgrade
```

Comment:
I think you can move all these function to ./tests/accel/upgrade/internal package.
Proposal:
You can add ./tests/accel/upgrade/internal/create package and ./tests/accel/upgrade/internal/delete package. usage example in code

delete.Workload()
create.Workload()
create.Service()
....